src/modules/featureInfo/i18n/featureInfo.provider.js

```diff
@@ -5,15 +5,19 @@ export const provide = (lang) => {
 				//the first part of the snake_case key should be the name of the related module
 				featureInfo_close_button: 'Close',
 				featureInfo_header: 'Object Information',
-				featureInfo_info: 'No information available.<br>Please click into the <b>map</b>.'
+				featureInfo_info: 'No information available.<br>Please click into the <b>map</b>.',
+				featureInfo_featureCollection_add_feature: 'Dieses Feature zur Sammlung hinzufügen',
```

Comment:
I guess, "Feature" is more a domain specific word (GIS), we should at leas consider "Objekt" as better alternative for standard user.